modules/sdk-core/src/bitgo/wallet/iWallets.ts

```diff
@@ -14,15 +14,24 @@ export interface GetWalletOptions {
   id?: string;
 }
 
-export interface GenerateMpcWalletOptions {
-  multisigType: 'onchain' | 'tss' | 'blsdkg';
+export interface GenerateBaseMpcWalletOptions {
+  multisigType: 'tss' | 'blsdkg';
```

Comment:
Why are we removing `onchain` if we're keeping it here https://github.com/BitGo/BitGoJS/pull/4083/files#diff-b9c562d50edf1eea34767e101d6fe3887bd184b940eb28e6d39abc6f900c77a6R175

Reply:
`onchain` isn't a valid MPC type, so it contradicts the type name to accept `onchain` (also likely a copy-past mistake when it was introduced)